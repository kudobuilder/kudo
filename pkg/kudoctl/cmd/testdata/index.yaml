apiVersion: v1
entries:
  mysql:
    - appVersion: "5.7"
      digest: aafe419e62065c5a3dc2f5d1458f1b5977137918236c3812ba90b5c7f6b8fb20
      maintainers:
        - email: nick@dischord.org
          name: Ken Sipe
      name: mysql
      operatorVersion: 0.1.0
      urls:
        - https://kudo-repository.storage.googleapis.com/0.7.0/mysql-0.1.0.tgz
    - appVersion: "5.8"
      digest: aafe419e62065c5a3dc2f5d1458f1b5977137918236c3812ba90b5c7f6b8fb20
      maintainers:
        - email: kensipe@gmail.com
          name: Ken Sipe
      name: mysql
      operatorVersion: 0.2.0
      urls:
        - https://kudo-repository.storage.googleapis.com/0.7.0/mysql-0.2.0.tgz
  redis:
    - appVersion: 5.0.1
      digest: 24012b833b8c864f24c03f5d792c986143b031c6783dd410538040d672813cd1
      maintainers:
        - email: djannot@mesosphere.io
          name: Denis Jannot
        - email: fabian@mesosphere.io
          name: Fabian Baier
      name: redis
      operatorVersion: 0.1.0
      urls:
        - https://kudo-repository.storage.googleapis.com/0.7.0/redis-0.1.0.tgz
generated: "2019-09-12T15:04:34.675941-05:00"