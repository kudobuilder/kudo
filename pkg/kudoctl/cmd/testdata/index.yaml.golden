apiVersion: v1
entries:
  zookeeper:
  - appVersion: 3.4.10
    digest: 050c5e67279c41e40e054d0e93d1361187ba8c9a289b5a00eb1c68cef4f0b18e
    maintainers:
    - email: avarkockova@mesosphere.com
      name: Alena Varkockova
    - email: runyontr@gmail.com
      name: Tom Runyon
    - email: kensipe@gmail.com
      name: Ken Sipe
    name: zookeeper
    operatorVersion: 0.1.0
    urls:
    - http://localhost/zookeeper-3.4.10_0.1.0.tgz
generated: "2019-10-25T00:00:00Z"
